tests/x11regressions/tracker/tracker_by_command.pm

```diff
@@ -17,12 +17,9 @@ use testapi;
 sub run() {
     my $self = shift;
     x11_start_program("xterm");
-    sleep 2;
-    wait_idle;
     script_run "tracker-search newfile";
-    sleep 5;
     assert_screen 'tracker-cmdsearch-newfile';
-    script_run "exit";
+    send_key 'ctrl-d';
```

Comment:
yes, I know. I think in this case it will even be safe because the "assert_screen" just before provides a synchronisation point but you know, people will copy-paste what you wrote. They *trust* you, this is dangerous - like trusting the "Richish English" ;-)